package.json

```diff
@@ -50,7 +50,7 @@
     "build:installer": "ts-node --transpile-only -P automation/tsconfig.json automation/run.ts build:installer",
     "package": "npm run build:fast && npm run build:standalone && npm run build:installer",
     "release": "ts-node --transpile-only -P automation/tsconfig.json automation/run.ts release",
-    "pretest": "npm run build",
+    "pretest": "npm run build && tsc -P tests/tsconfig.json --noEmit",
```

Comment:
Type checking for Typescript test files had been disabled when the `test` script was modified to use `ts-node/register/transpile-only` instead of `ts-node/register`.  I've learned that `resin-lint --typescript` performs Typescript linting **without** type checking.